src/Components/Modals/ActivationKeyWizard.js

```diff
@@ -100,23 +110,31 @@ const CreateActivationKeyWizard = ({
       id: 0,
       name: 'Name and Description',
       component: (
-        <SetNameAndDescriptionPage
-          name={name}
+        <NameAndDescriptionPage
+          activationKey={activationKey}
+          mode={isEditMode ? 'edit' : 'create'}
```

Comment:
Suggestion: Avoid casting to strings when a boolean can continue to suffice. Here, the mode is set to `'edit'` or `'create'` based on a boolean and then later converted back to a boolean (ex `mode === 'edit' ? ...`). Prefer passing as boolean the whole time